packages/fury-adapter-oas3-parser/lib/parser/oas/parseSecuritySchemeObject.js

```diff
@@ -13,27 +13,18 @@ const parseString = require('../parseString');
 
 const name = 'Security Scheme Object';
 const requiredKeys = ['type'];
-const unsupportedKeys = [
-  'bearerFormat', 'flows', 'openIdConnectUrl',
-];
+const unsupportedKeys = ['bearerFormat', 'openIdConnectUrl'];
 const isUnsupportedKey = R.anyPass(R.map(hasKey, unsupportedKeys));
-const passThrough = R.anyPass(R.map(hasKey, ['name', 'in', 'scheme']));
+const passThrough = R.anyPass(R.map(hasKey, ['name', 'in', 'scheme', 'flows']));
 
 const isApiKeyScheme = securityScheme => securityScheme.getValue('type') === 'apiKey';
 const isHttpScheme = securityScheme => securityScheme.getValue('type') === 'http';
+// const isOauth2Scheme = securityScheme => securityScheme.getValue('type') === 'oauth2';
```

Comment:
What's this comment for?

Reply:
This is the integration of oauth2 into security reminder. Should have been a TODO.